admin/class-yoast-form.php

```diff
@@ -537,36 +537,36 @@ public function media_input( $var, $label ) {
 
 		$id_field_id = 'wpseo_' . $var_esc . '_id';
 
-		echo '<span>';
-		echo '<input',
-			' class="textinput"',
-			' id="wpseo_', $var_esc, '"',
-			' type="text" size="36"',
-			' name="', esc_attr( $this->option_name ), '[', $var_esc, ']"',
-			' value="', esc_attr( $val ), '"',
-			' readonly="readonly"',
-			' /> ';
-		echo '<input',
-			' id="wpseo_', $var_esc, '_button"',
-			' class="wpseo_image_upload_button yoast-button yoast-button--secondary"',
-			' type="button"',
-			' value="', esc_attr__( 'Upload Image', 'wordpress-seo' ), '"',
-			' data-target-id="', esc_attr( $id_field_id ), '"',
-			disabled( $this->is_control_disabled( $var ), true, false ),
-			' /> ';
-		echo '<input',
-			' class="yoast-remove"',
-			' type="button"',
-			' value="', esc_attr__( 'Remove image', 'wordpress-seo' ), '"',
-			disabled( $this->is_control_disabled( $var ), true, false ),
-			' />';
-		echo '<input',
-			' type="hidden"',
-			' id="', esc_attr( $id_field_id ), '"',
-			' name="', esc_attr( $this->option_name ), '[', $var_esc, '_id]"',
-			' value="', esc_attr( $id_value ), '"',
-			' />';
-		echo '</span>';
+		echo '<div class="yoast-field-group yoast-field-group__upload">';
+			echo '<input',
+				' class="textinput yoast-field-group__inputfield"',
+				' id="wpseo_', $var_esc, '"',
+				' type="text" size="36"',
+				' name="', esc_attr( $this->option_name ), '[', $var_esc, ']"',
+				' value="', esc_attr( $val ), '"',
+				' readonly="readonly"',
+				' /> ';
+			echo '<input',
+				' id="wpseo_', $var_esc, '_button"',
+				' class="wpseo_image_upload_button yoast-button yoast-button--secondary yoast-button--upload"',
+				' type="button"',
+				' value="', esc_attr__( 'Upload Image', 'wordpress-seo' ), '"',
```

Comment:
We shouldn't use title case. Will change `Upload Image` to `Upload image` and create a new issue for all the other cases in the codebase.